pages/common/ibmcloud.md

```diff
@@ -0,0 +1,24 @@
+# ibmcloud
+
+> A command line tool for managing IBM Cloud apps and services.
+> More information: <https://ibm.biz/cli-docs>.
+
+- Show help for a specific subcommand (use without subcommand for general help):
+
+`ibmcloud help {{subcommand}}`
+
+- Show ibmcloud version:
+
+`ibmcloud version`
+
+- Update ibmcloud to the latest version:
+
+`ibmcloud update`
```

Comment:
Does this only update the `cli` tool or also other items related to IBM cloud(apps)?

Reply:
To the best of my knowledge (and according to official docs) this updates only the CLI. I can add that for clarity.

Reply:
I think when adding ``` ` ... ` ``` around it like we do in all pages when talking about commands, it's more clear that you're talking about the CLI itself.